empire.go

```diff
@@ -141,6 +145,10 @@ func (opts CreateOpts) Event() CreateEvent {
 
 // Create creates a new app.
 func (e *Empire) Create(ctx context.Context, opts CreateOpts) (*App, error) {
+	if err := e.MaintenanceMode; err != nil {
```

Comment:
having to boilerplate this out seems like it's going to be forgotten/easily missed with added functionality.  Is there anyway we can make this a function of setting up a new url/route possibly?  Maybe even grouping our URLs by 'read'/'write' (also based on the method used to call them).